line-bot-api.gemspec

```diff
@@ -20,7 +20,6 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version     = '>= 2.4.0'
 
   spec.add_development_dependency "addressable", "~> 2.3"
-  spec.add_development_dependency "bundler", "~> 1.11" if RUBY_VERSION < "2.3"
```

Comment:
Do not use RUBY_VERSION in line-bot-api.gemspec
https://sue445.hatenablog.com/entry/2016/09/02/121047